test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/DataServiceContextTests.cs

```diff
@@ -240,7 +242,7 @@ public void DefaultResolveTypeShouldReturnType()
         [TestMethod]
         public void DisposeShouldBeCalledOnResponseMessageForExecuteWithNoContent()
         {
-            DataServiceContext context = new DataServiceContext();
+            DataServiceContext context = new DataServiceContext().ReConfigureForNetworkLoadingTests();

```

Comment:
Since your `ReConfigureForNetworkLoadingTests` resets the dsc why not just reuse `testSubject` here?